src/components/ExceptionNotificationTest/ExceptionNotificationTest.tsx

```diff
@@ -0,0 +1,45 @@
+import { ErrorReporter } from 'lib/errors';
+import React, { ReactNode } from 'react';
+import Button from 'UI/Button';
+
+function generateUncaughtException() {
+  // @ts-ignore
+  // eslint-disable-next-line no-undef
+  return a_thing_that_doesnt_exist_on_purpose;
```

Comment:
Is there any reason for not throwing a new error here? (e. g. `throw new Error("...")`)?

Reply:
Oh, that makes more sense :)